docs/HackGuide.md

```diff
@@ -51,6 +51,14 @@
 - Domain / cert
   - Register a TLD in a subscription in the tenant
   - Add a wildcard cert stored in Key Vault
+    - Include CA bundle with wildcard cert in key vault by combining cert and ca-bundle files into a variable, then import value into the ingress secret.
```

Comment:
@AAkindele does this need to be base64 encoded?  we ran into some type of issue on this ...

Reply:
This is fine. The value for the app gateway cert if the one that needs to be base64 encoded.